test/e2e/helpers.go

```diff
@@ -71,3 +76,80 @@ func WaitForSuccessfulDeployment(t *testing.T, f *test.Framework, deployment app
 	t.Logf("Deployment %s has been initialized successfully\n", deployment.Name)
 	return nil
 }
+
+// createRuntimeObject creates a runtime object using the test framework
+func createRuntimeObject(f *test.Framework, ctx *test.TestCtx, obj runtime.Object) error {
+	return f.Client.Create(
+		context.TODO(),
+		obj,
+		&test.CleanupOptions{
+			TestContext:   ctx,
+			Timeout:       cleanupTimeout,
+			RetryInterval: cleanupRetryInterval,
+		})
+}
+
+// deleteRuntimeObject deletes a runtime object and wait for up to 1 minute for the
+// object to be removed from the cluster
+func deleteRuntimeObject(f *test.Framework, obj runtime.Object, name string, namespace string) error {
+	// Delete the runtime object
+	err := f.Client.Delete(
+		context.TODO(),
+		obj)
+	if err != nil {
+		return fmt.Errorf("Error deleting obj: %v", err)
+	}
+
+	// Confirm that the runtime object was deleted
+	namespacedName := types.NamespacedName{Name: name, Namespace: namespace}
+	result := pollUntilTrue(5*time.Second, 1*time.Minute, func() bool {
+		err = f.Client.Get(
+			context.TODO(),
+			namespacedName,
+			obj)
+		if apierrors.IsNotFound(err) {
+			return true
+		}
+		return false
+	})
+	if result {
+		return nil
+	}
+	return fmt.Errorf("Runtime object %v was never deleted", obj)
+}
+
+// createNamespace creates a namespace
+func createNamespace(f *test.Framework, ctx *test.TestCtx, namespace string) error {
+	obj := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}
+	// Create the operatorsource to download the manifests
+	return f.Client.Create(
```

Comment:
Please reuse `createRuntimeObject()` here?

Reply:
Addressed in the latest commit.